app/helpers/layout_helper.rb

```diff
@@ -40,7 +40,7 @@ def fetch_locations
   end
 
   def fetch_user
-    { current_user: User.current, user_dropdown: Menu::Manager.to_hash(:side_menu), impersonated_by: User.unscoped.find_by_id(session[:impersonated_by]) }
+    { current_user: User.current.to_json(only: [:id, :firstname, :lastname, :mail, :admin, :last_login_on, :name]), user_dropdown: Menu::Manager.to_hash(:side_menu), impersonated_by: User.unscoped.find_by_id(session[:impersonated_by]) }
```

Comment:
should be as_json to not break Layout.js

Reply:
Thx!